### ISSUE TITLE
Support following `apply from` links

### ISSUE THREAD

**Issue by BrettDuclos** [nebula-plugins/gradle-lint-plugin#84]
When adding the gradle-lint plugin to a project that is configured in a structure that has gradle broken out into script plugins, dependencies contained within are not being detected. 

For an example of a project that follows this convention see the Gradle project (https://github.com/gradle/gradle)

When gradle-lint is run against a project with this convention, I believe that none of these files are being consumed at all, and the generated report is missing most of the expected output.

To verify this, I have written the following test, showing this behavior.

```groovy
package com.netflix.nebula.lint

class ExternalGradleFilesSpec extends TestKitSpecification {

    def main = '''          
            public class Main {
                public static void main(String[] args) {
                }
            }
        '''

    def "Test dependencies in external gradle file"() {
        setup:
        File dependenciesFile = new File("$projectDir/gradle/dependencies.gradle")
        dependenciesFile.parentFile.mkdir()

        when:
        buildFile.text = """
            plugins {
                id 'nebula.lint'
                id 'java'
            }

            gradleLint.criticalRules = ['unused-dependency']

            repositories { mavenCentral() }

            apply from: "$dependenciesFile"
        """

        dependenciesFile.text = """
            dependencies {
                compile 'io.springfox:springfox-core:2.0.2'
            }
        """

        createJavaSourceFile(main)

        then:
        runTasksSuccessfully('compileJava', 'fixGradleLint')
        dependencies(buildFile) == ['io.springfox:springfox-core:2.0.2']
    }
}
```

**Comment by jkschneider** [comment]
@rspieldenner @DanielThomas FYI - this is implemented with 7.2.0
